files/image_config/role/role-config.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# role-config.sh
+#
+# Script to perform tasks which configure device based on its role as determined by its minigraph
+# To be run by role-config.service
+#
+
+# Disable DHCP relay service if device does not require it
+DHCP_RELAY_SERVICE_START_FILE=/etc/sonic/role/dhcp_relay
+DEVICE_ROLE=`sonic-cfggen -m /etc/sonic/minigraph.xml -v "minigraph_devices[minigraph_hostname]['type']"`
+
+if [ $DEVICE_ROLE == "ToRRouter" ]; then
+    echo "Device requires DHCP relay service. Ensuring start file exists..."
+    touch $DHCP_RELAY_SERVICE_START_FILE
```

Comment:
Can we do systemctl disable dhcp-relay.service instead of using additional files?